src/Aws/Laravel/AwsServiceProvider.php

```diff
@@ -28,7 +24,14 @@
  */
 class AwsServiceProvider extends ServiceProvider
 {
-    const VERSION = '1.1.0';
+    const VERSION = '1.5.0';
```

Comment:
I don't think you should upgrade the L4 version at all. Laravel 4 hasn't been supported by laravel at all since last year.